src/main/java/org/folio/rest/converter/PackagesConverter.java

```diff
@@ -80,4 +101,19 @@ private PackageCollectionItem convertPackage(PackageData packageData) {
       .withRelationships(EMPTY_PACKAGES_RELATIONSHIP);
   }
 
+  private Token convertToToken(TokenInfo packageToken) {
+    if(Objects.isNull(packageToken)){
```

Comment:
Minor comment -- is is possible to have a common method that can be used by both package and vendor/provider?